cluster/saltbase/salt/flannel/init.sls

```diff
@@ -0,0 +1,42 @@
+flannel-tar:
+  archive:
+    - extracted
+    - user: root
+    - name: /usr/local/src
```

Comment:
It has helper scrips and a README, feels more like a tiny git repo than an sbin or bin